pkgs/top-level/aliases.nix

```diff
@@ -116,6 +116,9 @@ mapAliases ({
   cv = progress; # added 2015-09-06
   d1x_rebirth = dxx-rebirth; # added 2018-04-25
   d2x_rebirth = dxx-rebirth; # added 2018-04-25
+  dart_dev = throw "Non-stable versions of Dart have been removed."; # added 2020-01-15
+  dart_old = dart; # added 2020-01-15
```

Comment:
Wouldn't this upgrade the dart version? Not sure if we want this.

Reply:
Well, it is either this or a `throw`, because `dart_old` will not work anymore.

Reply:
I think a throw would be better to not silently upgrade people.